tests/support/setup.js

```diff
@@ -30,11 +31,11 @@ define([
         );
       } else {
         type = 'mouse' + shortType;
-        e = document.createEvent('MouseEvent');
-        e.initMouseEvent(
-          type, true, true, null, null, 0, 0, 0, 0, false, false,
-          false, false, 0, null
-        );
+        e = new MouseEvent(type, {
+          bubbles: true,
+          cancelable: true,
+          buttons: buttons
+        });
```

Comment:
😭 We'll need to find a way to set the buttons property on IE's mouse event. I think it is only relevant for IE9, since IE10 uses `MSPointerEvent` and IE11 has native pointer events.

The need for a valid buttons property arises from proper button state checking during capture. 

Alternatively, with more and more functional tests, we may be able to skip the four capture related unit tests on IE9 (or in general).